step-by-step-general/01-requirements.md

```diff
@@ -7,10 +7,8 @@ You need to have the following to follow this guide:
     - Android 8 or later *
     - ≥1GB storage space
 
-- About 15 minutes of time (depends on your device)
-    - speedrun current record: 4:15
+- About 10 minutes of time (depends on your device)
     - flagship: estimated 5 ~ 10 min
-    - mid-tier: estimated 10 ~ 15 min
-    - low-tier: estimated 15 ~ 35 min
+    - mid to low tier: estimated 10 ~ 15 min
```

Comment:
I made these changes bec the current times seemed to reflect the old patching times, which have since drastically improved